manager/allocator/allocator.go

```diff
@@ -32,6 +33,13 @@ type Allocator struct {
 
 	// pluginGetter provides access to docker's plugin inventory.
 	pluginGetter plugingetter.PluginGetter
+
+	// DefaultAddrPool specifies default subnet pool for global scope networks
+	defaultAddrPool []*net.IPNet
+
+	// SubnetSize specifies the subnet size of the networks created from
```

Comment:
just subnet size for default subnet pool.  I thought comment has explanation so we can keep name simple. Is that ok with you?

Reply:
right, but does it signify the default size or the current size or the max size? I think that needs to be specified in the comment.

Reply:
there is only one subnet pool net mask length size. The address pool is named defaultAddrPool(because we use this as default if user doesnt mention subnet range). We don't have multiple values. As part of review comment I updated in the description part what it means.